cpp/src/arrow/compute/api_aggregate.h

```diff
@@ -66,7 +66,7 @@ struct ARROW_EXPORT MinMaxOptions : public FunctionOptions {
     /// Skip null values
     SKIP = 0,
     /// Any nulls will result in null output
-    OUTPUT_NULL
+    EMIT_NULL
```

Comment:
Can we simply change this? (is it public on the C++ side?) And it is changed for consistency with how it's called elsewhere?

Reply:
The next version will be 2.0.0, meaning it's ok to break APIs. Of course, we should try to minimize breakage, but this one aims to fix an inconsistency.